Makefile

```diff
@@ -16,7 +16,7 @@ endif
 all: backend frontend
 
 tools/golangci-lint: backend/go.mod backend/go.sum
-	cd backend && go build -o ./tools/golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint
+	GOBIN=`pwd`/backend/tools go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
```

Comment:
This is still showing the problem I had on Linux (WSL).

```bash
GOBIN=`pwd`/backend/tools go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
...
go: downloading github.com/quasilyte/go-ruleguard v0.3.19
go: downloading golang.org/x/exp/typeparams v0.0.0-20230224173230-c95f2b4c22f2
cd backend && ./tools/golangci-lint run
ERRO Running error: context loading failed: no go files to analyze 
make: *** [Makefile:22: backend-lint] Error 5
```

Reply:
I wonder what your `go version` is there? 

It's working for me in my WSL Ubuntu 22.04, so I wonder what the difference is.

```
$ uname -a
... 5.15.57.1-microsoft-standard-WSL2
$ go version
go version go1.20.6 linux/amd64
```

```
> wsl --version
WSL version: 0.66.2.0
Kernel version: 5.15.57.1
WSLg version: 1.0.42
MSRDC version: 1.2.3401
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.2283
```

Reply:
> The backend-lint is still not working for me in WSL.
> Approving nonetheless since the code LGTM and doesn't have regressions/issues AFAICS.

I'll move this discussion into an issue, and we can try to figure it out separately of this PR. https://github.com/headlamp-k8s/headlamp/issues/1404